test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] You did a good job working on setting up your first project at Microverse! 💯 However, according to the requirements, please make sure that this file is ignored by git. Ignored files shouldn't be present in your remote repository. 🤓 🛠️

Reply:
hi, how can I do that, am trying but its still present

Reply:
@Tracy-miranja, I can see that you have removed the file. Well done 👏